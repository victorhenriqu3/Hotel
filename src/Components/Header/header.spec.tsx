import { render, screen } from '@testing-library/react';
import { Header } from '.';

describe('Should render the Header correctly', () => {
	it('The title can only be capitalized', () => {
		render(<Header />);
		expect(screen.getByText(/ARGOS/)).toBeInTheDocument();
		expect(screen.getByText(/HOTEL/)).toBeInTheDocument();
	});
	it('The Cart must not contain any items', () => {
		render(<Header />);
		expect(screen.getByTestId(/CartIndicator/)).toBeInTheDocument();
	});
});
