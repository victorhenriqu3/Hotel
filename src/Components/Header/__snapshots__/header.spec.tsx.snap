// Vitest Snapshot v1

exports[`Should render the Header correctly > The title can only be capitalized 1`] = `
<p
  class="chakra-text css-1piseh9"
>
  ARGOS
</p>
`;
